src/controllers/home.ts

```diff
@@ -1,9 +1,22 @@
 import {Request, Response} from 'express'
+import {Pagination} from '../lib/pagination'
 
 export class HomeController {
 	public index() {
 		return (request: Request, response: Response) => {
-			response.render('index')
+			let searchResults: any = {
+				combinedResults: [],
+				page: 0,
+				results: [1, 2, 3, 4, 5, 6, 7, 8, 9, 0],
+				size: 10,
+				totalResults: 10,
+			}
```

Comment:
this is hardcoded as placeholder until we integrate the API call to learning catalogue